Applications/src/detect-cardiac-phases.cc

```diff
@@ -0,0 +1,132 @@
+/*
+ * Medical Image Registration ToolKit (MIRTK)
+ *
+ * Copyright 2008-2015 Imperial College London
+ * Copyright 2008-2013 Daniel Rueckert, Julia Schnabel
+ * Copyright 2013-2015 Andreas Schuh
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mirtk/Options.h"
+#include "mirtk/IOConfig.h"
+
+#include "mirtk/BaseImage.h"
+#include "mirtk/GenericImage.h"
+#include "mirtk/GaussianBlurring.h"
+
+using namespace mirtk;
+
+char *cine_name = NULL;
```

Comment:
In IRTK, people used global variables like this. But there is no justification for this. Could you please make these local variables within the `main` function scope where they are used? Also, use `const char *` instead of `char *`, or even `string` (from `std`, but imported into `mirtk` namespace; hence, `std::string` not necessary). For easy cases, I suggest `const char *` but moving the declaration to where the variable is used the first time.